challenge.md

```diff
@@ -6,19 +6,29 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+git clone <ssh url>
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+git log
+
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
 
+git show <commit id>
```

Comment:
The command we were looking for was `git log`. `git show` is a lot more general, and provides a lot of additional info that we're not necessarily looking for. I won't take points off, though.